packages/terra-demographics-banner/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Revert components updated to use `injectIntl` to interface with `react-intl's` `intl` context.
```

Comment:
This should just be "Components updated to use `injectIntl` to interface with `react-intl's` `intl` context." again.

Reply:
Hey @tbiethman , fixed it. Thank you.